instructure/canvas-android
# [Student][MBL-14301] Add confetti for on-time submission

To test:
 - Ensure celebrations are enabled at the account level and that the user does not have the "Disable celebrations" flag enabled
 - Make a submission to an assignment that is not past-due. Confetti should be shown.
 - Make a submission to a past-due assignment. Confetti should not be shown.
 - After enabling the "Disable celebrations" flag in the user settings on web and clearing the app's cache, making an on-time submission should not show confetti.
